conbench/entities/benchmark_result.py

```diff
@@ -184,6 +185,9 @@ def _dump(self, benchmark_result):
             "run_id": benchmark_result.run_id,
             "batch_id": benchmark_result.batch_id,
             "timestamp": benchmark_result.timestamp.isoformat(),
+            "language": benchmark_result.context.tags.get(
```

Comment:
We should include `context` dict (that includes `language` field) in benchmark results json the same way as we include `tags` dict.

Reply:
Conbench already has a concept of benchmark language. It expects it to be called `benchmark_language`, and that it is sent in the `context`.

- https://github.com/conbench/conbench/search?q=benchmark_language
- https://github.com/conbench/conbench/search?q=display_language

![Uploading Screen Shot 2022-07-19 at 12.35.13 PM.png…]()

Reply:
You can also filter by the benchmark language. Hope that helps, cheers!

<img width="1281" alt="Screen Shot 2022-07-19 at 12 43 14 PM" src="https://user-images.githubusercontent.com/878798/179825430-cf8a3b93-156c-4e6e-8848-d8dcf49a2ec1.png">

<img width="1239" alt="Screen Shot 2022-07-19 at 12 42 28 PM" src="https://user-images.githubusercontent.com/878798/179825423-9ac808a7-3a61-431e-9e3c-041f89dfecb5.png">

Reply:
Sorry for the tardy response. Great thought about using the contexts endpoint. I am largely interacting with this via the api so a join on the context id would be just a fruitful here and avoids overloading a benchmark result with _all_ of the context. Right now benchmarks returns a link to the context. Certainly I can process that to extract the id but I wonder if it might be useful to return `context_id` here. I can modify the PR to do that instead which is maybe a bit cleaner?